src/utils/saveFeaturedArtworks.ts

```diff
@@ -0,0 +1,103 @@
+import "dotenv/config"
+import { Connection, createConnection, getMongoRepository } from "typeorm"
+import { databaseConfig } from "../config/database"
+import { Collection } from "../Entities"
+import metaphysics from "../lib/metaphysics"
+
+/**
+ * Cycle through all collections in DB to fetch and save featuredArtworks
+ */
+export const saveFeaturedArtworks = async () => {
+  let connection: Connection | undefined
+
+  try {
+    const connectionArgs = databaseConfig()
+    connection = await createConnection(connectionArgs)
+
+    if (connection.isConnected) {
+      const repository = getMongoRepository(Collection)
+      const collections = await repository.find()
+
+      for (const collection of collections) {
+        const { slug } = collection
+        try {
+          const collectionWithImages = await attachFeaturedArtworks(collection)
+          await repository.update({ slug }, collectionWithImages)
+          console.log("Successfully updated: ", slug)
+        } catch (e) {
+          console.log(`Error saving ${slug}`, e)
+        }
+      }
+    } else {
+      throw new Error("Could not connect to database")
+    }
+  } finally {
+    /* tslint:disable:no-unused-expression */
+    connection && connection.close()
+    /* tslint:disable:no-unused-expression */
+  }
+}
+
+/**
+ * For a given collection, attach top 3 artworks ranked by merchandisability
+ */
+export const attachFeaturedArtworks = async collection => {
+  try {
+    const featuredArtworks = await getFeaturedArtworks(
+      `{
+        marketingCollection(slug: "${collection.slug}") {
```

Comment:
I don't think we need to do something about it now but here's my concern with the current request path for resolving this query

1. KAWS sends the initial request to Metaphysics 
2. Metaphysics uses schema stitching to resolve `marketingCollection` and makes a request to KAWS
3. Metaphysics makes a request to gravity to resolve the search query for artworks
4. KAWS gets the response back

That's 4 roundtrips. We could cut that to one by using the following query instead
```graphql
{
 filterArtworks(filters...) {
 }
}
```